src/main/java/com/dataguise/saas/JWTAuthenticationFilter.java

```diff
@@ -0,0 +1,35 @@
+package com.dataguise.saas;
+
+import com.dataguise.saas.service.JWTService;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.filter.GenericFilterBean;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class JWTAuthenticationFilter extends GenericFilterBean {
+
+    @Override
+    public void doFilter(ServletRequest request,
```

Comment:
**Reporter**: Checkstyle    
    **Rule**: com.puppycrawl.tools.checkstyle.checks.FinalParametersCheck
    **Severity**: ERROR
    **File**: /work/src/main/java/com/dataguise/saas/JWTAuthenticationFilter.java L19
    
    Parameter request should be final.
    
*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a988800699>*